Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -407,55 +632,329 @@ void UpdateTabIcons()
 		protected virtual void SetTabIcon(TabLayout.Tab tab, FileImageSource icon)
 		{
 			tab.SetIcon(ResourceManager.IdFromTitle(icon, ResourceManager.DrawableClass));
+
+			if (tab.Icon != null)
+			{
+				tab.SetIcon(ADrawableCompat.Wrap(tab.Icon));
+				tab.Icon.Mutate();
+			}
+
+			this.SetIconColorFilter(tab);
 		}
 
 		void UpdateBarBackgroundColor()
 		{
-			if (_disposed || _tabLayout == null)
+			if (IsDisposed)
 				return;
 
-			Color tintColor = Element.BarBackgroundColor;
+			if (IsBottomTabPlacement)
+			{
+				Color tintColor = Element.BarBackgroundColor;
 
-			if (Forms.IsLollipopOrNewer)
+				if (Forms.IsLollipopOrNewer)
+				{
+					if (tintColor.IsDefault)
+						_bottomNavigationView.SetBackgroundColor(Color.Default.ToAndroid());
```

Comment:
is this the default color on Android? we should use that

Reply:
Good catch